CONTRIBUTING.md

```diff
@@ -1,77 +1,116 @@
-# Contributing to Direct State Transfer (DST)
+# Contributing to the Hyperledger Lab Perun
 
 Want to contribute? That's great! Any contribution is welcome, e.g.
 documentation, bug reports, feature request, issues, blog posts,
 tutorials, feature implementations, etc. You can contribute code or
 documentation through the standard GitHub pull request model. For large
 contributions we do encourage you to file a ticket in the GitHub issue
-tracking system prior to any development to coordinate with the DST
+tracking system prior to any development to coordinate with the Perun
 development team early in the process. Coordinating up front helps to
 avoid frustration later on.
 
-*Note that this repository only contains the documentation of DST. For
-issues / pull requests related to source code please refer to the
-[dst-go](https://github.com/direct-state-transfer/dst-go) repository.*
+*Note that this repository only contains the documentation of the Perun
+project. For issues / pull requests related to source code please refer to the
+[perun-node](https://github.com/hyperledger-labs/perun-node) repository.*
 
 ## Contribution Guideline
 
 * We use the GitHub [issue
-  tracker](https://github.com/direct-state-transfer/dst-doc/issues) and
+  tracker](https://github.com/hyperledger-labs/perun-doc/issues) and
   the [github-flow](http://scottchacon.com/2011/08/31/github-flow.html)
-  branching model for dst-doc.
+  branching model for perun-doc.
 
-* When you want to submit a patch to any DST project, you must [sign a
-  CLA (once) and sign your commits](#sign-the-cla-and-sign-your-work).
+* When you want to submit a patch to any Hyperledger Labs project, you
+  must [sign your commits](#sign-your-work).
```

Comment:
```suggestion
* When you want to submit a patch, you must [sign your commits](#sign-your-work).
```

As it is in the [`Contributing.md`](https://github.com/direct-state-transfer/perun-node/blob/develop/CONTRIBUTING.md#contribution-guideline) of `perun-node` repo.